cluster/gce/trusty/node.yaml

```diff
@@ -139,6 +139,8 @@ script
 		--configure-cbr0=true \
 		--cgroup-root=/ \
 		--system-container=/system \
+                --runtime-container=/docker-daemon \
+                --resource-container=/kubelet \
```

Comment:
Why aren't we using spaces instead of tabs? Spaces are portable across editors. I can update tabs to be spaces in the entire file if that's ok with you @andyzheng0831.